sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SymbolicExecution/TooManyInternalStatesException.cs

```diff
@@ -19,10 +19,26 @@
  */
 
 using System;
+using System.Runtime.Serialization;
 
 namespace SonarAnalyzer.Helpers.FlowAnalysis.Common
 {
+    [Serializable]

```

Comment:
You are right the scope is small now but I'd rather fix the code than mark a FP on this case on Next.